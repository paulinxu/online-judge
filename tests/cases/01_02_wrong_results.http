###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 413
# content-type: application/json
# date: Thu, 25 Jul 2024 15:32:54 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"","result":"Wrong Answer","time":0}],"created_time":"2024-07-25T15:32:54.548774711Z","id":0,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World.\"); }","user_id":0},"updated_time":"2024-07-25T15:32:54.730807897Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 413
# content-type: application/json
# date: Thu, 25 Jul 2024 15:32:54 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"","result":"Runtime Error","time":0}],"created_time":"2024-07-25T15:32:54.732106991Z","id":1,"result":"Runtime Error","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { panic!(\"Hello World!\"); }","user_id":0},"updated_time":"2024-07-25T15:32:54.900612028Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 413
# content-type: application/json
# date: Thu, 25 Jul 2024 15:32:54 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Error","time":0},{"id":1,"info":"","result":"Waiting","time":0}],"created_time":"2024-07-25T15:32:54.902436520Z","id":2,"result":"Compilation Error","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!{}\"); }","user_id":0},"updated_time":"2024-07-25T15:32:54.938296699Z"}
