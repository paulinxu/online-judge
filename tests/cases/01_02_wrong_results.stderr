[2024-07-25T15:32:53Z INFO  actix_server::builder] starting 16 workers
[2024-07-25T15:32:53Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2024-07-25T15:32:54Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000159
[2024-07-25T15:32:54Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 413 "-" "-" 0.182366
[2024-07-25T15:32:54Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 413 "-" "-" 0.168774
[2024-07-25T15:32:54Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 413 "-" "-" 0.036120
